app/src/main/java/com/portfolio/tasky/usecases/OfflineModeHandler.kt

```diff
@@ -0,0 +1,7 @@
+package com.portfolio.tasky.usecases
+
+import androidx.appcompat.widget.AppCompatTextView
+
+interface OfflineModeHandler {
```

Comment:
The usecase would be the "Network Change Receiver", this one just modifies the UI. So this can be implemented directly on the UI

Reply:
Since all this impl does was change UI so have implemented in the EntryActivity itself.